src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/idam/IdamTestService.java

```diff
@@ -146,4 +149,14 @@ private CreateUserRequest createSolicitorRequest(String username, String passwor
             password
         );
     }
+
+    private void createUser(CreateUserRequest createUserRequest) {
+        //recommended delay from IDAM team to stop intermittent auth failures
```

Comment:
If this recommendation is a temporary solution, we can live for a while. This should not be the permanent solution. It's going to effect us in terms of time and money in the long run.